docs/sharding.md

```diff
@@ -0,0 +1,107 @@
+# Multiple Azure SignalR Service instances Support
+Currently we add support for configuring multiple SignalR Service instances under **persistent** mode. You can distribute your clients to multiple SignalR service instances and send messages to multiple instances as if to one instance. 
+
+<!-- TOC -->
+
+- [Multiple Azure SignalR Service instances Support](#multiple-azure-signalr-service-instances-support)
+  - [Usage Scenarios](#usage-scenarios)
+  - [Configuration method](#configuration-method)
+  - [Route algorithm](#route-algorithm)
+    - [Default behavior](#default-behavior)
+    - [Customization](#customization)
+    - [Implicitly pass HttpContext to endpoint router](#implicitly-pass-httpcontext-to-endpoint-router)
+
+<!-- /TOC -->
+
+## Usage Scenarios
```

Comment:
Keep consistent: Upper case for each word for all ## headers, or only the first word